### PR TITLE
Test PR for new lint checks

### PR DESCRIPTION
**Pull request by davidkocnar**
- Testing new lint checks on CI
- This PR should throw a few errors

### CODE REVIEW

File: app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.thefuntasty.androidprojecttemplate
+
+import android.app.Service
+import android.content.Intent
+import android.os.IBinder
+
+class TestService : Service() {
+
+
```
Comment (thefuntasty-ops) [inline app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/android-project-template/blob/b0765a4c13fb0e70984828239cb03f4d8d6ea3cf/app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt#L9"></span>Needless blank line(s)</td>
    </tr>
  </tbody>
</table>

File: app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    //TODO: Test Service only to check new lint checks, but tpyos (typos) shouldnt be a reported problem
```
Comment (thefuntasty-ops) [inline app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/android-project-template/blob/b0765a4c13fb0e70984828239cb03f4d8d6ea3cf/app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt#L10"></span>This comment contains text that has been defined as forbidden in detekt.</td>
    </tr>
  </tbody>
</table>

Comment (thefuntasty-ops) [inline app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/android-project-template/blob/b0765a4c13fb0e70984828239cb03f4d8d6ea3cf/app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt#L10"></span>Missing space after //</td>
    </tr>
  </tbody>
</table>

File: app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.thefuntasty.androidprojecttemplate
+
+import android.app.Service
+import android.content.Intent
+import android.os.IBinder
+
+class TestService : Service() {
```
Comment (thefuntasty-ops) [inline app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/android-project-template/blob/b0765a4c13fb0e70984828239cb03f4d8d6ea3cf/app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt#L7"></span>The <code>&lt;service&gt; com.thefuntasty.androidprojecttemplate.TestService</code> is not registered in the manifest</td>
    </tr>
  </tbody>
</table>

File: app/src/main/res/layout/fragment_main.xml [hunk 4]
```diff
@@ -23,6 +24,13 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            android:text="Hello test of hardcoded string"/>
```
Comment (thefuntasty-ops) [inline app/src/main/res/layout/fragment_main.xml]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/android-project-template/blob/b0765a4c13fb0e70984828239cb03f4d8d6ea3cf/app/src/main/res/layout/fragment_main.xml#L32"></span>Hardcoded string “Hello test of hardcoded string”, should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>

**Comment by thefuntasty-ops** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Branch name should have <code>release/</code>, <code>hotfix/</code>, <code>fix/</code>, <code>housekeep/</code> or <code>feature/</code> prefix.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by davidkocnar** [comment]
The test was successful. Ktlint, detekt and Android lint checks on CI are working.

**Pull request closed without merging**